### PR TITLE
Added scripts, modified requirements, added subpackages

### PR DESCRIPTION
**Pull request by csxeba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,31 @@
...
 try to make it learn inside the dream.
 
+## Grund integration and minimal documentation
+
+The environment, which we are trying to learn is Match from grund.
+
+To access it, grund has to be installed:
+
+```
+pip install https://github.com/csxeba/grund
```
Comment (gaborvecsei) [inline README.md]: If we use Python3 it should be `pip3 install git+https://github.com/csxeba/grund.git`

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+numpy
+tensorflow
+keras
+https://github.com/csxeba/grund
```
Comment (gaborvecsei) [inline requirements.txt]: Syntax here is almost the same as in the readme: `git+https://github.com/csxeba/grund.git#egg=grund`

https://pip.readthedocs.io/en/1.1/requirements.html

**Review by gaborvecsei — CHANGES_REQUESTED** [review]
Nice start!!! 💯

Just a few minor changes requested

**Comment by csxeba** [comment]
Done and done!

**Review by gaborvecsei — APPROVED** [review]

**Comment by gaborvecsei** [comment]
You can **Squash & Merge** 👍

**Comment by csxeba** [comment]
Didn't squash tho

**Pull request merged**